example/hands-on/services/bucky_hands_on/pc/parts/github_top.yml

```diff
@@ -0,0 +1,6 @@
+search_bar:
```

Comment:
why do you commit this file?

Reply:
I commit all files which are created in this example.